### ISSUE TITLE
package only works if app uses default name

### ISSUE THREAD

**Issue by vesper8** [mitulgolakiya/laravel-api-generator#29]
I was just having trouble with another composer package which required me to rename my app to a different namespace.

And then I started having trouble with this package. It seems that because you use this statement in your controller

```
use App\Http\Controllers\Controller;
```

That this package will only work for applications that use the default namespace. I have renamed mine and the only way I could get your package to work was to rename that use statement and add my namespace.

This is the first time I am faced with such a conflict... is there something I am missing?

**Comment by mitulgolakiya** [comment]
@vesper8, 
You are right, above use statement is static as of now. so if you change root namespace then it will create trouble. I will look into it and will find out a way to fix it in next release.

**Comment by mitulgolakiya** [comment]
@vesper8,
I just released a new version with custom app namespace support. Refer documentation here to make it working: https://github.com/mitulgolakiya/laravel-api-generator#using-with-custom-application-namespace

**Comment by vesper8** [comment]
You had fixed this apparently but now the problem is back. Using the dev-master I am still running into many cases where I have had to manually change the namespace since I am using a custom namespace different than "App". Specifically, the requests are being generated with 'App/Http/Requests' even though I have set the correct namespace in the generator.php config file under 'namespace_request' so it is being ignored.

The base controller is also having the same issue as what I initially opened this issue about.

On a side note, it appears that the path_views from the generator.php is also being ignored. I tried having my views generated in a subfolder of resources/views/API/ and getting the error InvalidArgumentException in FileViewFinder.php line 137:
View [networks.index] not found.

but it works fine if I leave it to the default
